translatepy/translators/yandex.py

```diff
@@ -1,19 +1,21 @@
-from time import time
-from json import loads
-from random import randint
-from os.path import dirname, abspath
+"""
+Yandex Translate
+
+This implementation was made specifically for translatepy from 'Zhymabek Roman', based on 'Anime no Seka' version.
+
+"""
+
 from typing import Union
 from urllib.parse import quote
+import uuid
 
-import pyuseragents
-from safeIO import TextFile
 from requests import get, post
 
 from translatepy.models.languages import Language
-from translatepy.utils.utils import convert_to_float
-from translatepy.utils.annotations import Tuple, Dict
+from translatepy.utils.annotations import Tuple
+from translatepy.utils.lru_cacher import timed_lru_cache
 
-FILE_LOCATION = dirname(abspath(__file__))
+# TODO: Implement getting a list of supported languages by querying Yandex Translate API request
```

Comment:
I found an API method that gives out all Yandex Translate supported languages, I think it will be more reliable. Tomorrow then I will try to implement